modelskill/comparison/_collection.py

```diff
@@ -235,18 +214,36 @@ def __repr__(self):
             out.append(f"{type(value).__name__}: {key}")
         return str.join("\n", out)
 
-    def __getitem__(self, x) -> Comparer:
+    @overload
+    def __getitem__(self, x: slice) -> ComparerCollection:
```

Comment:
It says slice, but can I also provide a list of names? e.g. cc[["stn3", "stn1"]] ? (like pandas or xarray)

Reply:
no, not yet

Reply:
No it is possible, but I can not get type hints to work 100%

In this case it should be a `Comparer`🤔
![image](https://github.com/DHI/modelskill/assets/614215/ce28d6e5-38d3-4f86-a2a8-bdbe954fba7c)